src/helpers/utility.ts

```diff
@@ -487,6 +494,22 @@ function isArcApp(appId?: string) {
   return [constants.constants.ARC_APP_ID, constants.constants.INV_ARC_APP_ID].includes(_appId)
 }
 
+function logoLargeWithName(appId?: string) {
+  return isArcApp(appId) ? ArcLogoLarge : OpenBridgeLogoLarge
+}
+
+function logoSmallWithName(appId?: string) {
+  return isArcApp(appId) ? ArcLogoSmall : OpenBridgeLogoSmall
+}
+
+function logoWithName(hasSubNav?: boolean, appId?: string) {
+  return hasSubNav ? logoSmallWithName(appId) : logoLargeWithName(appId)
+}
+
+function logoSymbolOnly(appId?: string) {
+  return isArcApp(appId) ? ArcLogoSymbol : OpenBridgeLogoSymbol
+}
```

Comment:
yeah, I thought about that but then it meant unraveling `isArcApp` and `getAppId` for what amounts to a band-aide so that ARC DIAN's site can be updated with the correct logos.